client/src/hooks/datafiles/mutations/useMove.ts

```diff
@@ -0,0 +1,113 @@
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { useSelectedFiles } from 'hooks/datafiles';
+import { useMutation } from '@tanstack/react-query';
+import { apiClient } from 'utils/apiClient';
+import Cookies from 'js-cookie';
+
+export async function moveFileUtil({
+  // Establish variables and their types per TypeScript
```

Comment:
You can remove these comments

Reply:
Gotcha. I think I had put these in as mental notes for myself instead of for everyone.
I removed the superfluous comments and kept the more important ones.